.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,15 @@
+# Source: https://github.com/marketplace/actions/pre-commit
+name: pre-commit
```

Comment:
Thoughts on calling this `lint` instead of `pre-commit` as a workflow name?  Since we just happen to use pre-commit to implement linting.

Reply:
Yeah I am comfortable with that